src/tests/bignum_test.nr

```diff
@@ -790,3 +790,10 @@ fn test_expressions() {
     assert(wx_constrained.limbs == wx.limbs);
 }
 
+#[test]
```

Comment:
I guess missing a test case where Field is large enough so it will produce 2 limbs